lib/manageiq/appliance_console/message_configuration_client.rb

```diff
@@ -82,6 +82,8 @@ def fetch_ca_cert_from_server
       def fetch_from_server(src_file, dst_file)
         return if file_found?(dst_file)
 
+        FileUtils.mkdir_p(dst_file.dirname) unless dst_file.dirname.directory?
```

Comment:
This appears to always be a `Pathname` (callers: [`fetch_truststore_from_server`](https://github.com/ManageIQ/manageiq-appliance_console/blob/master/lib/manageiq/appliance_console/message_configuration_client.rb#L71), [`fetch_ca_cert_from_server`](https://github.com/ManageIQ/manageiq-appliance_console/blob/master/lib/manageiq/appliance_console/message_configuration_client.rb#L74-L78))